src/media/js/app_list.js

```diff
@@ -12,13 +12,15 @@ define('app_list',
     }
 
     function setTrays(expanded) {
+        var $toggle = $('.app-list-filters-expand-toggle');
         // Handle tray state and populate preview thumbs.
         // Preserve the tray expand state in localStorage.
         if (expanded !== undefined) {
             expand = expanded;
         }
         $('.app-list').toggleClass('expanded', expanded);
-        $('.app-list-filters-expand-toggle').toggleClass('active', expand);
+        $toggle.toggleClass('active', expand);
+        $toggle.addClass('show');
```

Comment:
We have existing uses of 'show' elsewhere and it's quite clear. I don't see a benefit of prefixing it for here.